blog/sign-your-code.md

```diff
@@ -0,0 +1,68 @@
+# Sign your code
+
+Do you know you can sign your code in Git using a GPG key? A lot of programmers doesn't know their can sign their Git commits using a signature created by themself.
+
+## Introduction
+
+You can use any name or email when you create a Git commit, but you can't sign a commit with a GPG key that doesn't belong to you.
+
+### What is GPG?
+
+The GNU Privacy Guard is an implementation of the OpenPGP standard and allows you to encrypt and sign your data and communications.
+
+This is fully integrated with Git and you can automatically sign your code with it.
+
+You can create a GPG key following [this guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key) in GitHub, remember that you should register it in [GitHub](https://github.com/settings/keys), or [GitLab](https://gitlab.com/-/profile/gpg_keys).
+
+## Should I sign every commit?
+
+There is a discussion about this, because Linus Torvalds says that when you're signing a Git tag 🏷️, you're validating all the commits in the release, signed or not. If you **automatically** sign every commit in the repository, the source of the signature lose its sense.
```

Comment:
```suggestion
There is a discussion about this, because Linus Torvalds says that when you're signing a Git tag 🏷️, you're validating all the commits in the release, signed or not. If you **automatically** sign every commit in the repository, the source of the signature loses its sense.
```